robots/src/robot.cpp

```diff
@@ -3,18 +3,28 @@
 #include <services.hpp>
 
 #include "../include/lyfco_e1600_robot.hpp"
+#if ROBOT_PLATFORM_Sabo
 #include "../include/sabo_robot.hpp"
+#endif
```

Comment:
no need for compile time checks

Reply:
This would mean that every robot, also the one without a display, need to compiles lvgl which would be a pain for them during development.